Tutorial_1a_Convolving_Images.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    theta_input = hdu.header['BMAJ']*3600 # in arcsec
</pre><p>Add a note that the beam widths are usually in deg, and a quick suggestions on how to check (which could just be print out the beam)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>